src/proc_fuse.c

```diff
@@ -213,6 +213,25 @@ static uint64_t get_memlimit(const char *cgroup, bool swap)
 	return memlimit;
 }
 
+/*
+ * dirname() behaves differently on musl than glibc, so implement the bits we
+ * need (the path in this case is always absolute and never ends in a `/`)
+ */
+static char *cgroup_parent(char *cgroup)
+{
+	if (strcmp(cgroup, "/") == 0) {
```

Comment:
This can just be *cgroup == '/'

Reply:
Oops, actually just realised it has to be `strcmp()`, `*cgroup == '/'` will always be true haha.